### PR TITLE
Security profile

### PR DESCRIPTION
**Pull request by julianbermudez**
This pull request enables the selection of security in Fast RTPS.

### CODE REVIEW

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Linux/90/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Windows/84/)

File: CMakeLists.txt [hunk 1]
```diff
@@ -30,6 +30,7 @@ option(UAGENT_CED_PROFILE "Build CedMiddleware profile." ON)
 option(UAGENT_DISCOVERY_PROFILE "Build Discovery profile." ON)
 option(UAGENT_P2P_PROFILE "Build P2P discovery profile." ON)
 option(UAGENT_LOGGER_PROFILE "Build logger profile." ON)
+option(UAGENT_SECURITY_PROFILE "Build logger profile." OFF)
```
Comment (BorjaOuterelo) [inline CMakeLists.txt]:
```suggestion
option(UAGENT_SECURITY_PROFILE "Build security profile." OFF)
```

File: cmake/SuperBuild.cmake [hunk 2]
```diff
@@ -80,13 +80,15 @@ if(UAGENT_FAST_PROFILE)
...
                 -DTHIRDPARTY:BOOL=ON
+                -DSECURITY:BOOL=${UAGENT_SECURITY_PROFILE}
             DEPENDS
                 fastcdr
             )
         list(APPEND _deps fastrtps)
     endif()
 endif()
 
+
```
Comment (BorjaOuterelo) [inline cmake/SuperBuild.cmake]:
```suggestion
```

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Linux/93/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Windows/87/)

**Pull request merged**